docs/src/pages/components/tree-view/IconExpansionTreeView.tsx

```diff
@@ -19,12 +19,35 @@ const useStyles = makeStyles({
   },
 });
 
+const useContentStyles = makeStyles((theme) =>
```

Comment:
To be honest, requiring developers to do this seems like a regression that we could fix. How about we add these styles in the `StyledTreeItemContent` using:
```
[`& .${treeItemClasses.iconContainer}`]: { ... },
[`& .${treeItemClasses.label}`]: { ... },
```

Reply:
I see this is the last comment unresolved. Let me know if it is not clear @siriwatknp :)

Reply:
done